geode-redis/src/integrationTest/java/org/apache/geode/redis/CommandPipeliningIntegrationTest.java

```diff
@@ -93,6 +98,42 @@ public void whenPipelining_commandResponsesAreNotCorrupted() {
     assertThat(mockSubscriber.getReceivedMessages()).isEqualTo(expectedMessages);
   }
 
+
+  @Test
+  public void should_returnResultsOfPipelinedCommands_inCorrectOrder() {
+    Logger logger = LogManager.getLogger("org.apache.geode.redis.internal");
+    Configurator.setAllLevels(logger.getName(), Level.getLevel("DEBUG"));
+    FastLogger.setDelegating(true);
+
+    Jedis jedis = new Jedis("localhost", server.getPort(), 10000000);
+    int NUMBER_OF_COMMANDS_IN_PIPELINE = 100;
+    int numberOfPipeLineRequests = 1000;
+
+    do {
+      Pipeline p = jedis.pipelined();
+      for (int i = 0; i < NUMBER_OF_COMMANDS_IN_PIPELINE; i++) {
+        p.echo(String.valueOf(i));
+      }
+
+      List<Object> results = p.syncAndReturnAll();
+
+      verifyResultOrder(NUMBER_OF_COMMANDS_IN_PIPELINE, results);
+      numberOfPipeLineRequests--;
+    } while (numberOfPipeLineRequests > 0);
+
+    jedis.flushAll();
+    jedis.close();
+  }
+
+  private void verifyResultOrder(int NUMBER_OF_COMMAND_IN_PIPELINE, List<Object> results) {
```

Comment:
NUMBER_OF_COMMAND_IN_PIPELINE looks like a constant, but isn't. It should be something like "numberOfCommands" or "numberOfCommandsInPipeline".

Reply:
made final.  Thanks!